src/Router/Entity/Param/Enum.php

```diff
@@ -62,13 +60,19 @@ public static function __set_state($state)
 
     public function getName(): string
     {
-        return parent::getName(); // TODO: Change the autogenerated stub
+        return parent::getName();
```

Comment:
why the heck is this here? I don't think i really needed to override anything with the getName method...

Reply:
Donno, you did it :)